src/cli/init.ts

```diff
@@ -231,6 +239,42 @@ export const initCommand = new Command()
         packageName,
         threadBuilder,
       });
+    } else if (circuitType === "jolt") {
+      // Jolt.
+      const packageName = await input({
+        message: "Jolt Package Name:",
+        default: circuitName
+          .toLowerCase()
+          .replace(/[- ]/g, "_")
+          .replace(/[^a-zA-Z0-9_]+/, "")
+          .replace(/_+/g, "_"),
+        validate: (input): boolean | string => {
+          if (input.length === 0) {
+            return "You must specify a package name.";
+          }
+          if (!/^[a-zA-Z0-9_]+$/.test(input)) {
+            return "Package names must only contain alphanumeric characters and underscores.";
+          }
+          return true;
+        },
+      });
+      const commitmentScheme: "hyperkzg" | "zeromorph" = await select({
+        message: "Commitment Scheme:",
+        default: "hyperkzg",
+        choices: [
+          { name: "Zeromorph", value: "zeromorph" },
+          { name: "HyperKZG", value: "hyperkzg" },
```

Comment:
I lean towards alphabetizing unless there's a strong reason not to.

```suggestion
          { name: "HyperKZG", value: "hyperkzg" },
          { name: "Zeromorph", value: "zeromorph" },
```